en/docs/integrations/integration-concepts.md

```diff
@@ -44,6 +44,7 @@ Choreo supports the following statements:
 
  - **Log**: Logs an event with an information statement or an error that occurs in your Choreo integration.
  - **Variable**: Declares a variable and a value of a preferred data type (`string`, `int`, etc.) to use later in the logical process of the Choreo integration.
+ - **Data Mapping**: Allows to create variables by visually mapping one or more variables, see [Data Mapping](../references/data-mapping.md).
```

Comment:
Can we divide this sentence into 2 ?

Suggestion: "Allows to create variables by visually mapping one or more variables. For more information, see [Data Mapping](../references/data-mapping.md).

Reply:
Fixed